### PR TITLE
More en_US localisation changes

### PR DESCRIPTION
**Pull request by dominichayesferen**
Mainly centered around the current newest noticeable feature, being the customisable keyboard shortcuts, but does have a few minor consistency fixes and tweaks.

### CODE REVIEW

File: floorp.ftl [hunk 1]
```diff
@@ -571,29 +571,29 @@ workspace-icon-chill =
  .label = Private
 
 # CSK
-floorp-CSK-title = Custom Shortcut Keys
-floorp-CSK-description = Customize the shortcut keys of { -brand-short-name }. Floorp provides more 80 shortcut keys. Reboot { -brand-short-name } to apply these settings. Duplicates do not work.
+floorp-CSK-title = Custom keyboard shortcuts
+floorp-CSK-description = Customize { -brand-short-name }'s keyboard shortcuts. Floorp provides more than 80 customizable keyboard shortcuts - restart { -brand-short-name } to apply these settings. Duplicate keyboard shortcuts will not work.
 disable-fx-actions =
- .label = Disable Firefox's default shortcut keys
+ .label = Disable built-in keyboard shortcuts
```
Comment (surapunoyousei) [inline floorp.ftl:577]: In fact, This feature does not disable, Workspaces shortcut key.

Is this correct?

Comment (dominichayesferen) [reply floorp.ftl:577]: Ah, in that case...

Comment (surapunoyousei) [reply floorp.ftl:577]: I think both are good.

I'm not native. You can choice.

Comment (dominichayesferen) [reply floorp.ftl]: Nah, built-in implies Floorp's shortcuts too so people would assume such and consider it a bug if I keep it that way.

**Review by surapunoyousei — APPROVED** [review]

**Comment by surapunoyousei** [comment]
Thanks!!

**Comment by surapunoyousei** [comment]
BTW, What do you think of new English localization repository style?

We separated from Floorp repo.

**Comment by dominichayesferen** [comment]
> BTW, What do you think of new English localization repository style?
> 
> We separated from Floorp repo.

It's alright, I suppose. Would probably be better to merge this with the other localisations repository though, and maybe also house brand.ftl here in the process somehow (since that has localisations via its own separate file, technically).

**Comment by surapunoyousei** [comment]
Hmm.

Ok I will try it.

**Pull request merged**